src/App.js

```diff
@@ -74,6 +75,11 @@ class App extends Component {
   constructor(props){
     super(props);
 
+    const welcomeShown = JSON.parse(localStorage.getItem("welcomeShown"));
```

Comment:
it may break the app if the item in the local storage is not a json. Need to catch the error